generators/app/templates/README.md

```diff
@@ -13,10 +13,9 @@
 ## What?
 
 ## Why?
-
 <% if (isOpenSource) { _%>
+
 ## License
 
 This software is released under the MIT license. See [the license
-file](LICENSE) for more details.
-<%_ } %>
+file](LICENSE) for more details.<%_ } %>
```

Comment:
This moves closing bracket to this line to prevent markdownlint errors on new project generation